ROADMAP.md

```diff
@@ -0,0 +1,226 @@
+* HTML layout for guide docs
+* HTML layout update for API docs to include refs to guide docs
+
+#### :two: Guide and Topic Docs
+
+A _very_ close second to the work on the documentation tooling is the work on
+identifying and creating new guides and topic documentation. These two terms
+come up a lot, to clarify what they mean, here's an except from our [getting
+started][ref-getting-started] guide:
+
+> 1. **Guide** documents explain processes to help the reader learn a concept
+> in service of their larger goal. Usually a guide has the reader build
+> something — a little webserver, or CLI — and explains the concept it's trying
+> to convey using examples from the readers experience with that code. Guides
+> are great for introducing new concepts in a comfortable way, by letting the
+> reader "simulate" the process of what development will be like using those
+> concepts.
+> 2. **Topic** documents explain concepts to help the reader make a decision.
+> They are a great place for "deep dive" information, and to handle anything
+> that's fairly intricate.
+> 3. **Reference** documents explain capabilities to help the reader achieve
+> their goal. API documentation is the primary example of this.
+
+Node.js currently has **reference** documentation, but because of the lack of
+**topic** and **guide** documentation, it ends up having to repeat itself a
+lot. Sometimes the lack of a dedicated place for this kind of documentation
+means that reference docs will attempt to include guides or topic docs inline —
+this usually doesn't end up working well in the long run.
+
+The Docs WG has identified and created a few guides already, some of which are
+the subject of the [first](#one-pulling-guides-into-nodejsnode) task.
+*However*, we need help filling in the gaps — both in identifying needed docs,
+as well as in executing on them.
+
+It's important to note that this documentation applies as much to topics
+*internal* to the project as it does to our externally facing API — we need
+guides and overviews for the architecture of Node, as well as how to
+collaborate on the project.
+
+##### Subtasks
+
+Want to pitch in? Look here! If someone's already working on the task, see
+if they need help.
+
+* Overviews:
+  * "Encoding", to be used by the Buffer and FS docs.
+  * "Blocking vs. non-blocking."
+  * Syscall documentation, to be used primarily by FS, but throughout the codebase.
+  * Internal: Node.js Architecture — **@eljefedelrodeodeljefe** [#71](https://github.com/nodejs/docs/issues/71)
+  * Internal: Initialization process
+  * Internal: Timers
+  * Internal: Event loop
+  * Internal: Signals
+  * Internal: Docs WG Process — **@chrisdickinson**
+* Guides:
+  * Internal: New Collaborator guide — **@nodejs/inclusivity**, **@ashleygwilliams** [nodejs/inclusivity#96](https://github.com/nodejs/inclusivity/issues/96)
+  * Internal: Move "cutting releases" into these guides.
+  * Streams: For Authors
+  * Streams: For Consumers
```

Comment:
Thanks — I've added your name. Mind creating a tracking ticket for this? I'll merge the ticket number in after this is merged.